.gitlab-ci.yml

```diff
@@ -83,6 +88,30 @@ build-in-docker-static:
   variables:
     EXTRA_CMAKE_FLAGS: "-DBUILD_SHARED_LIBS=OFF"
 
+upload-artifacts:
+  stage: upload
+  extends: .in-prplmesh-builder
+  script:
+    # the owncloud password is stored in the file pointed by "$CI_OWNCLOUD_PASSWORD".
+    # We can only store the password in it, since spaces are not supported in Gitlab masked variables.
+    # We symlink the file to .netrc and pre-pend to it to avoid manipulating the password itself
```

Comment:
I still don't understand why we symlink instead of copy the file...

Reply:
There is no particular reason for a symlink over a copy. 
If it feels more "natural", I can make a copy instead.
Is there a particular advantage to making a copy?